pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1577,6 +1577,12 @@
         "4.2.0.0": "4.5.0"
       }
     },
+    "System.Data.Odbc": {
+      "InboxOn": {},
+      "AssemblyVersionInPackageVersion": {
+        "4.0.1.0": "4.5.0"
```

Comment:
We should bump the version back down to 4.0.0.0, since it hasn't ever shipped. This is done in src/System.Data.Odbc/dir.props.